vck/src/commonMain/kotlin/at/asitplus/wallet/lib/iso/cborSerializer.kt

```diff
@@ -43,11 +43,10 @@ internal object CborCredentialSerializer {
 }
 
 @OptIn(ExperimentalSerializationApi::class)
-val cborSerializer by lazy {
-    Cbor {
+val vckCborSerializer by lazy {
+    Cbor(from = at.asitplus.crypto.datatypes.cose.io.cborSerializer) {
```

Comment:
I would not remove the explicit configuraiton of the serializer in this library

Reply:
Here I disagree, since this one is a COSE-complaint serializer and this is exactly what we want here. Everything beyond COSE compliance is explicitly configured, just as it was before.
What it does lack is a proper API documentation in kmp-crypto